ssz/tools/io.py

```diff
@@ -0,0 +1,150 @@
+from collections.abc import (
+    Mapping,
+    Sequence,
+)
+
+from eth_utils import (
+    decode_hex,
+    encode_hex,
+)
+
+from ssz.sedes import (
+    BaseSedes,
+    Boolean,
+    ByteList,
+    ByteVector,
+    Container,
+    List,
+    UInt,
+    Vector,
+)
+from ssz.sedes.serializable import (
+    BaseSerializable,
+    MetaSerializable,
+)
+
+
+class FormattedDictIO:
+    def __init__(self, int_as):
+        self.int_as = int_as
```

Comment:
See my comment about using an interface rather than hard coding the various ways things can be formatted.  We have a defined set of basic types so it seems reasonable to just define the interface for dumping/loading and then to pass an instance of that interface into this API.